theblues/plans.py

```diff
@@ -0,0 +1,50 @@
+from collections import namedtuple
+import datetime
+
+from theblues.errors import (
+    log,
+    ServerError,
+)
+from theblues.utils import (
+    ensure_trailing_slash,
+    make_request,
+)
+
+Plan = namedtuple('Plan', 'url plan created_on description price')
+
```

Comment:
Ah, I am misunderstanding named tuples--but it looks like in the [namedtuple docs](https://docs.python.org/dev/library/collections.html#collections.namedtuple) it should be

```python
Plan = namedtuple('Plan', ['url',  'plan', 'created_on', 'description', 'price'])
```